internal/mocks/inmem/ccipreader_inmem.go

```diff
@@ -122,14 +122,20 @@ func (r InMemoryCCIPReader) GasPrices(
 	panic("implement me")
 }
```

Comment:
And then in the plugin for consensus we want the `(name, chain, address)` pairs observed by at least `F` followers?